src/utils/installableUtils.ts

```diff
@@ -39,18 +39,18 @@ import extensionPointRegistry from "@/extensionPoints/registry";
 import { getContainedExtensionPointTypes } from "@/utils/recipeUtils";
 
 /**
- * Returns true if installable is an UnavailableRecipe
+ * Returns true if installable is an UnavailableMod
  * @param installable the installable
- * @see UnavailableRecipe
+ * @see UnavailableMod
  */
-export function isUnavailableRecipe(
+export function isUnavailableMod(
   installable: Installable
-): installable is UnavailableRecipe {
+): installable is UnavailableMod {
   return "isStub" in installable && installable.isStub;
 }
 
 /**
- * Returns true if the installable is a singleton extension, not a recipe.
+ * Returns true if the installable is a singleton extension, not a mod.
```

Comment:
We're going to be renaming all usages of the word "installable" -> "mod", so this would read

> Returns true if the mod is a singleton extension, not a mod.

So I suggest something like:
```suggestion
 * Returns true if the mod is a standalone mod definition, instead of a mod definition.
```

See this part of the notion https://www.notion.so/pixiebrix/Proposal-Domain-Modelling-and-Naming-9f7c8ffc0ed4458599a67536d74bdc3a?pvs=4#1ab1df73769949c79c7b8ff48a251a58 for the discussion about`StandaloneModDefinition`

Reply:
Now that I think about it, we probably don't have to do this now. We haven't introduced `StandaloneModDefinition` yet.

Let's leave this feedback for future work.

Reply:
I was going off off this:

> StandaloneModDefinition: formerly CloudExtension